packages/web-react/src/components/Container/stories/Container.stories.tsx

```diff
@@ -23,6 +24,13 @@ const meta: Meta<typeof Container> = {
         defaultValue: { summary: 'false' },
       },
     },
+    size: {
+      control: 'select',
+      options: [...Object.values(SizesExtended)],
+      table: {
+        defaultValue: { summary: SizesExtended.XLARGE },
+      },
+    },
   },
   args: {
```

Comment:
Please add the size here as well. Sorry, I missed this one before.
```suggestion
  args: {
    children: <DocsBox>Container</DocsBox>,
    isFluid: false,
    size: 'xlarge',
  }
```